src/server/migrations/20200317235251_units.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('units', function(table) {
+    table
+      .increments('id')
+      .primary()
+      .unsigned()
+      .notNullable();
+    table
+      .varchar('name')
+      .unique()
+      .notNullable();
+  });
```

Comment:
Please add these fields created_at,updated_at, and deleted_at.

Reply:
Do you think that we should add 'created_at, updated_at and deleted_at' fields in units table? because in https://dbdiagram.io/d/5e58dfd24495b02c3b878ba4 db diagram table shows only 2 fields, id and name???

Reply:
We are going to update our DB diagram, please wait for it.